Dockerfile

```diff
@@ -113,17 +113,24 @@ COPY . /code/timemachine/
 WORKDIR /code/timemachine/
 RUN pip install --no-cache-dir -e . && rm -rf ./build
 
-# Container with only cuda runtime, half the size of the timemachine_cuda_dev container
-FROM nvidia/cuda:12.4.1-runtime-ubuntu20.04 as timemachine
+# Container with only cuda base, half the size of the timemachine_cuda_dev container
+# Need to copy curand/cudart as these are dependecies of the Timemachine GPU code
+FROM docker.io/nvidia/cuda:12.4.1-base-ubuntu20.04 as timemachine
 ARG LIBXRENDER_VERSION
 ARG LIBXEXT_VERSION
 RUN (apt-get update || true) && apt-get install --no-install-recommends -y libxrender1=${LIBXRENDER_VERSION} libxext-dev=${LIBXEXT_VERSION} \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
-COPY --from=timemachine_cuda_dev /opt/ /opt/
```

Comment:
This was previously copying `/opt/nvidia` which is about 1.5Gb for nvidia-compute which we never use.